src/pages/Home/Home.jsx

```diff
@@ -1,97 +1,123 @@
+            />
           </div>
-        </ul>
-      </nav>
 
-      <section className="container-img">
-        <img className="main-image" src="/assets/ecommerce-main.jpg" alt="" />
+          <div className="best-card">
+            <p className="card-heading">Best sellers</p>
 
-        <button className="button primary">   <Link
-          className="login"
-          to="/pages/product-listing/product-listing.html"
-        >
-          Shop Now
-        </Link></button>
-
-     
-      </section>
-      <div className="sub-heading">
-        <h2 className="h2">Shop by Category</h2>
+            <img
+              className="img-card adjust"
+              src="../assets/best-seller-nav.jpg"
+              alt=""
+            />
+          </div>
+        </div>
       </div>
-      <div className="card-container">
-        <div className="men-card">
-          <p className="card-heading">Men</p>
 
-          <img className="img-card adjust" src="/assets/men-nav.webp" alt="" />
+      <div className="category-row2">
+        <div className="sub-heading">
+          <h2 className="h2">Currently Trending</h2>
         </div>
+        <div className="card-container">
+          <div className="men-card">
+            <p className="card-heading">Pet supplies</p>
+
+            <img
+              className="img-card adjust"
+              src="/assets/pet-supplies.jpg"
+              alt=""
+            />
+          </div>
 
-        <div className="women-card">
-          <p className="card-heading">Women</p>
+          <div className="women-card">
+            <p className="card-heading">Gaming accessories</p>
 
-          <img
-            className="img-card adjust"
-            src="/assets/women-nav.webp"
-            alt=""
-          />
-        </div>
+            <img
+              className="img-card adjust"
+              src="/assets/gaming-accessories.jpg"
+              alt=""
+            />
+          </div>
 
-        <div className="kids-card">
-          <p className="card-heading">Kids</p>
+          <div className="kids-card">
+            <p className="card-heading">Health and Personal Care</p>
 
-          <img className="img-card adjust" src="/assets/kids-nav.webp" alt="" />
-        </div>
+            <img
+              className="img-card adjust"
+              src="/assets/health-personal-care.jpg"
+              alt=""
+            />
+          </div>
 
-        <div className="best-card">
-          <p className="card-heading">Best sellers</p>
+          <div className="best-card">
+            <p className="card-heading">Home Decor</p>
 
-          <img
-            className="img-card adjust"
-            src="../assets/best-seller nav.jpg"
-            alt=""
-          />
+            <img
+              className="img-card adjust"
+              src="../assets/home-decor.jpg"
+              alt=""
+            />
+          </div>
         </div>
       </div>
+
+      <Footer />
```

Comment:
You could add `Footer` directly in the `App.js` file, outside the Routes. No need to add it to every Screen.